lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -43,7 +43,8 @@ def last_rpm
       {'release' => release, 'rpm' => rpm}
     end
 
-    rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last['rpm']
+    rpm = rpms.sort { |a,b| a['release'].to_i <=> b['release'].to_i }.last
+    rpm['rpm'] if rpm
```

Comment:
actually after testing File.symlink() does not like nil.  How was this working before?  very strange